code-name-taran/src/com/spaghettisoft/component/hangman/WordToGuess.java

```diff
@@ -0,0 +1,109 @@
+package com.spaghettisoft.component.hangman;
+
+import java.io.IOException;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.File;
+
+public class WordToGuess {
+
+	private static final String FILE_NAME = "dict.txt.txt";
+	protected static String[] AVAILABLE_WORDS;

```

Comment:
You don't need this field to be static. Also only constants ("final variables") should be named with all caps.